src/style.css

```diff
@@ -1,3 +1,87 @@
-body {
+body,
+html {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  font-size: 0  ;
+}
+
+li {
+  position: relative;
+  margin: 0 10px;
+  display: inline-block;
+  /* background-color: #f28; */
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)